lib/ledger_sync/adaptors/quickbooks_online/account/operations/create.rb

```diff
@@ -0,0 +1,40 @@
+require 'ledger_sync/adaptors/quickbooks_online/account/mapping'
+
+module LedgerSync
+  module Adaptors
+    module QuickBooksOnline
+      module Account
+        module Operations
+          class Create < Operation::Create
+            class Contract < LedgerSync::Adaptors::Contract
+              params do
+                required(:ledger_id).value(:nil)
+                required(:name).filled(:string)
+                required(:account_type).filled(:string)
+              end
+            end
+
+            private
+
+            def operate
+              response = adaptor.upsert(
+                resource: 'account',
+                payload: local_resource_data
+              )
+
+              resource.ledger_id = response.dig('Id')
+              success(response: response)
+            end
+
+            def local_resource_data
+              {
+                'Name' => resource.name,
+                'AccountSubType' => Mapping::ACCOUNT_TYPES[resource.account_type]
```

Comment:
Why are you mapping the account_type attribute to AccountSubType? Those are two separate fields, so ideally we'd have the values for both. For example

Account Type -> Bank
Account Sub Type -> Checking

Reply:
If we can do value for both, thats even better. QBO requires one of these. I'll update it to support both separately.